lib/bundler/plugin.rb

```diff
@@ -197,7 +197,16 @@ def load_plugin(name)
       # done to avoid conflicts
       path = index.plugin_path(name)
 
+      if insert_index = Bundler.rubygems.load_path_insert_index
```

Comment:
we have to do something, since it is totally valid for a gem to have a require path that isn't lib